### PR TITLE
RA-1064 Added delete classes functionality

### PR DESCRIPTION
**Pull request by pgutkowski**

### CODE REVIEW

File: test/unit/controllerSpec.js [hunk 1]
```diff
@@ -48,5 +50,15 @@ describe('Concept dictionary controllers', function() {
     		  [{name: 'Anatomy', description: 'Anatomic sites / descriptors', uuid: 'ad491c7a-c2cc-11de-8d13-0010c6dffd0f'},
                {name: 'Procedure', description: 'Describes a clinical procedure', uuid: 'bd490bf4-c2cc-11de-8d13-0010c6dffd0f'}]);
     });
+    
+    it('should send delete request at uuid adress extracted from selection map', function() {
+    	//scope.adress equals Util.getOpenmrsUrl()
+        $httpBackend.expectDELETE(scope.adress+'/ws/rest/v1/conceptclass/ad491c7a-c2cc-11de-8d13-0010c6dffd0f?').respond("DELETE SUCCESS")
```
Comment (rkorytkowski) [inline test/unit/controllerSpec.js]: Have you seen https://docs.angularjs.org/api/ngMock/function/angular.mock.inject ? Just change to inject(function(Util) { ... }) in line 54

File: app/js/controllers.js [hunk 2]
```diff
@@ -8,6 +9,20 @@ conceptDictControllers.controller('ClassesListCtrl', ['$scope', 'loadClasses', '
...
+	//deletes selected classes
+	$scope.deleteSelected = function(){
+	    angular.forEach($scope.selected, function(key,value){
+	        if(key){
+	        	ClassesService.deleteClass({uuid : value});
+	        }
+	            
+	    });
+	    //reloads the page ($route.reload doesn't get updated list)
+	    location.reload();
```
Comment (rkorytkowski) [inline app/js/controllers.js]: Is the whole page reloaded? If yes, then maybe it's better to just do ClassesService.getAll().then(function(data) {
$scope.classes = data });

**Comment by rkorytkowski** [comment]
Overall looks good. I'd just do 2 small fixes.

**Comment by rkorytkowski** [comment]
Please address my comments and update the code to the latest so it can be merged. Feel free to push the merged code to upstream without opening a pull request.

**Comment by pgutkowski** [comment]
Thanks, these changes definitely would be an upgrade. 
I don't have access to my computer right now, I will do it tomorrow morning.

**Pull request merged**